colossus-docs/src/main/scala/RedisServiceExample.scala

```diff
@@ -0,0 +1,46 @@
+// #example
+import java.util.concurrent.ConcurrentHashMap
+import akka.actor.ActorSystem
+import akka.util.ByteString
+import colossus.IOSystem
+import colossus.protocols.redis._
+import colossus.protocols.redis.server.{Initializer, RedisServer, RequestHandler}
+import colossus.service.Callback
+import colossus.service.GenRequestHandler.PartialHandler
+
+object RedisServiceExample extends App {
+
+  implicit val actorSystem = ActorSystem()
+  implicit val ioSystem = IOSystem()
+
+  val db = new ConcurrentHashMap[String, String]()
+
+  RedisServer.start("example-server", 9000) {
+    new Initializer(_) {
+      override def onConnect = new RequestHandler(_) {
+        override def handle: PartialHandler[Redis] = {
+          case Command("GET", args) =>
```

Comment:
Just a comment, but I know at one point we used to have extractors for each command, which would have made this cleaner `case Get(key)`.  I think that was all removed when we created the `RedisClient` interface (which in general was a good move), but maybe we should think about bringing back the extractors.  I'd also not be opposed to a macro or code-generation tool for those since it is a pain to write/update stuff for each command in multiple places.